lib/facter/rhsm_repos.rb

```diff
@@ -1,11 +0,0 @@
-Facter.add('rhsm_repos') do
```

Comment:
Instead of removing the facts they might just output the chunk data from the all-in-one rhsm fact (backwards compatibility)